codec-http/src/main/java/io/netty/handler/codec/http/multipart/HttpPostMultipartRequestDecoder.java

```diff
@@ -1467,7 +1467,7 @@ private boolean skipOneLine() {
         valueStart = HttpPostBodyUtil.findNonWhitespace(sb, colonEnd);
         valueEnd = HttpPostBodyUtil.findEndOfString(sb);
         headers.add(sb.substring(nameStart, nameEnd));
-        String svalue = sb.substring(valueStart, valueEnd);
+        String svalue = sb.substring(valueStart, Math.max(valueStart, valueEnd));
```

Comment:
@matteobertozzi should we just make this `StringUtil.EMPTY_STRING` when `valueStart >= valueEnd` ?

So something like this:

```
String svalue == valueStart >= valueEnd ? StringUtil.EMPTY_STRING : sb.substring(valueStart, valueEnd))
```
WDYT ?

Reply:
yeah, that's better

Reply:
@matteobertozzi please adjust and push.